src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -2719,9 +2720,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
 
             Dim options = DirectCast(tree.Options, VisualBasicParseOptions)
-            Dim IOperationFeatureFlag = InternalSyntax.FeatureExtensions.GetFeatureFlag(InternalSyntax.Feature.IOperation)
-
-            Return If(IOperationFeatureFlag Is Nothing, False, options.Features.ContainsKey(IOperationFeatureFlag))
+            Dim IOperationFeatureFlag = VisualBasic.Language.Feature.IOperation.GetFeatureFlag
+            Return VisualBasic.Language.Feature.IOperation.IsAvailable(options)
+            'Return If(IOperationFeatureFlag Is Nothing, False, options.Features.ContainsKey(IOperationFeatureFlag))
```

Comment:
Please revert the changes to this method so that `GetLanguageVersion()` does not need to support prototype features.

Reply:
The majority of this Pull Request is aimed at centralising and unifying all of the separate functions the check to see if a feature is available, into one or two core methods.
That `IsPrototype` method is useful, once the feature `IOperation` has release version. 
```vbnet
<Extension>
Private Function IsPrototype(feature As Feature) As Boolean
    Select Case feature
        Case Feature.IOperation
            Return True
     End Select
     Return False
End Function
' to
<Extension>
Private Function IsPrototype(feature As Feature) As Boolean
    Select Case feature
     End Select
     Return False
End Function
```
and update `GetLanguageVersion`
```vbnet
Case Feature.InferredTupleNames
    Return LanguageVersion.VisualBasic15_3
Case Feature.IOperation
    Return LanguageVersion.VisualBasic17_0'     
```
None of the usage code (like above) as the change. It just works.

Reply:
Changing how `IOperation` is handled in `CheckFeatureAvailability` is outside the scope of #13228 which is limited to merging the implementations of `CheckFeatureAvailability` from `Parser` and `Scanner`. Moreover, the existing implementation for `IOperation` works, without affecting methods such as `GetLanguageVersion()`, so the change is unnecessary. Please revert the `IOperation` changes.